packages/apps/src/configs/evm/SupportedMixers.ts

```diff
@@ -9,6 +10,15 @@ export const getNativeCurrencySymbol = (chainID: number): string => {
   return 'Unknown';
 };
 
+export const getEVMChainName = (chainID: number): string => {
```

Comment:
Should this function and other helper functions be moved to a different file?

Reply:
we can keep them in the meantime